lib/srv/desktop/rdp/rdpclient/src/client.rs

```diff
@@ -1168,7 +1168,7 @@ fn create_config(params: &ConnectParams, pin: String) -> Config {
         no_server_pointer: false,
         autologon: true,
         pointer_software_rendering: false,
-        performance_flags: PerformanceFlags::default()
+        performance_flags: PerformanceFlags::DISABLE_CURSOR_SHADOW
```

Comment:
Are we losing any other default flags by doing this?

Shoudln't we do `PerformanceFlags::default() | PerformanceFlags::DISABLE_CURSOR_SHADOW`?

Reply:
We should, I don't know why I assumed that default is empty. It's fixed.